Desktop App uninstall (#6129)
mattermost/docs: source/install/desktop-app-install.rst

- macOS App Store: ``/Users/<username>/Library/Containers/Mattermost/Data/Library/Application Support/Mattermost``
- Linux: ``~/.config/Mattermost``

.. note::

>   If you choose to uninstall the desktop app, you must manually remove local configuration data from the applicable location noted above.

Reviewer: ```suggestion
  Local configuration data is not automatically removed when uninstalling the desktop app. If you wish to remove all data, you must manually remove the files from the applicable location noted above.
```

- macOS App Store: ``/Users/<username>/Library/Containers/Mattermost/Data/Library/Application Support/Mattermost``
- Linux: ``~/.config/Mattermost``

.. note::

  - Local configuration data is not automatically removed when uninstalling the desktop app. If you wish to remove all data, you must manually remove the files from the applicable location noted above.

>   - Prior to uninstalling, you can choose to log out of any active sessions. Active sessions may also be terminated from another Mattermost session by going to **Profile > Security > View and Logout of Active Sessions**, and selecting **Log Out**. Desktop app sessions are labeled as **Native Desktop App**.

Reviewer 2: ```suggestion
  - Prior to uninstalling, you can choose to log out of any active sessions. You can terminate active sessions from another Mattermost session in **Profile > Security > View and Logout of Active Sessions**, then select **Log Out**. Desktop app sessions are labeled as **Native Desktop App**.
```
